README.md

```diff
@@ -11,6 +11,8 @@ The plugin provides tasks for uploading, downloading, running and managing [Goog
 
 To use the App Engine plugin, include in your build script:
 
+    apply plugin: 'java'
+    apply plugin: 'war'
```

Comment:
potentially, using this plugin, users can use an 'ear'

```
apply plugin: 'war' // or 'ear'
```

Reply:
updated